packages/rx_bloc_cli/example/README.md

```diff
@@ -5,23 +5,31 @@
 | Path | Contains |
 | ------------ | ------------ |
 | `lib/base/` | Common code used on more than one **feature** in the project. |
+| `lib/base/app/` | The root of the application and Environment configuration. |
 | `lib/base/common_blocs/` | Global [BLoCs][rx_bloc_info_lnk]|
 | `lib/base/common_ui_components/` | Reusable widgets (buttons, controls etc) |
 | `lib/base/common_use_cases/` | Global UseCases |
+| `lib/base/data_sources/` | All data sources are placed here. |
+| `lib/base/data_sources/domain/` | Data sources,  relating to a specific domain |
+| `lib/base/data_sources/domain/counter/` | Data sources,  relating to the counter domain |
```

Comment:
The counter feature is a reference implementation, so we don't need to mention it in the documentation